src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -77,6 +77,8 @@ object PushdownUtilities {
         PushdownFilter("max" + colName.capitalize, value.toString)
       case LessThanOrEqual(colName, value) =>
         PushdownFilter("max" + colName.capitalize, value.toString)
+      case StringStartsWith(colName, value) =>
```

Comment:
I think we should add ` | StringStartsWith(_, null)` into the condition at the line 67 as well?